Mathlib/Topology/Algebra/UniformConvergence.lean

```diff
@@ -93,9 +92,44 @@ instance [CommGroup β] : CommGroup (α →ᵤ β) :=
 instance [CommGroup β] : CommGroup (α →ᵤ[𝔖] β) :=
   Pi.commGroup
 
+instance {M : Type*} [SMul M β] : SMul M (α →ᵤ β) := Pi.instSMul
+
+instance {M N : Type*} [SMul M N] [SMul M β] [SMul N β] [IsScalarTower M N β] :
+    IsScalarTower M N (α →ᵤ β) :=
+  Pi.isScalarTower
+
+instance {M N : Type*} [SMul M β] [SMul N β] [SMulCommClass M N β] :
+    SMulCommClass M N (α →ᵤ β) :=
+  Pi.smulCommClass
+
+instance {M : Type*} [Monoid M] [MulAction M β] : MulAction M (α →ᵤ β) := Pi.mulAction _
+
+instance {M : Type*} [Monoid M] [AddMonoid β] [DistribMulAction M β] :
+    DistribMulAction M (α →ᵤ β) :=
+  Pi.distribMulAction _
+
+instance [Semiring R] [AddCommMonoid β] [Module R β] : Module R (α →ᵤ β) :=
+  Pi.module _ _ _
+
 instance [Semiring R] [AddCommMonoid β] [Module R β] : Module R (α →ᵤ β) :=
   Pi.module _ _ _
```

Comment:
Isn't this one a duplicate?

Reply:
I'm going to close this one as I'm redoing it (and the lemmas below) the right way in another PR.